app/model.js

```diff
@@ -50,4 +50,8 @@ exports.updateArticle = (id, inc_votes)=>{
 
 exports.deleteFromComments = (id)=>{
     return db.query(`DELETE FROM comments WHERE comment_id = $1 RETURNING *`,[id])
+}
+
+exports.selectAllUsers = ()=>{
+    return db.query(`SELECT * FROM users`)
```

Comment:
Ideally in the model we should be chaining a .then and returning rows so that the controller is just receiving the array of users not the entire response from the query.